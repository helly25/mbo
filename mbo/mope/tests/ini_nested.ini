[person]
id=0
name=Hugo

[person.contact]
phone=1234

[person.contact:1]
phone=2345

[person:1]
id=1
name=Freddy

[person:1.contact]
phone=3456

[person:1.contact:1]
phone=4567
