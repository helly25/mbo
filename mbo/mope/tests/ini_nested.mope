{{#person:"\n"}}
Name: {{name}}
Id: {{id}}
{{#contact}}
Phone: {{phone}}
{{/contact}}
{{/person}}
