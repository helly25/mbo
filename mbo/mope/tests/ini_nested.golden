Name: Hugo
Id: 0
Phone: 1234
Phone: 2345

Name: Freddy
Id: 1
Phone: 3456
Phone: 4567
